lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -345,8 +345,17 @@ sub engine_workit {
     my $tmpdir = "$pooldir/tmp";
     mkdir($tmpdir) unless (-d $tmpdir);
 
+    # create and configure the process including how to stop it again
     my $child = process(
-        sub {
+        set_pipes                => 0,                   # disable additional pipes for process communication
+        internal_pipes           => 0,                   # disable additional pipes for retrieving process return/errors
+        _default_kill_signal     => -POSIX::SIGTERM(),
+        _default_blocking_signal => -POSIX::SIGKILL(),
+        max_kill_attempts        => 1,                   # stop the process by sending SIGTERM 1 time …
+        sleeptime_during_kill    => 10,                  # … and give processes 10 seconds to handle it ……
+        kill_sleeptime           => 0,                   # … and wait not any longer …
+        blocking_stop            => 1,                   # … before sending SIGKILL
+        code                     => sub {
```

Comment:
I think a more comment style is to set all the options after the sub but this is also ok for me

Reply:
Really? When a function takes multiple arguments and one of them is a `sub` I usually prefer if the `sub` is passed as last argument.

Reply:
I meant to write "a more common style", as in: "What I have seen more in code". I don't care personally